src/sage/plot/hyperbolic_regular_polygon.py

```diff
@@ -22,9 +22,10 @@
 from sage.rings.cc import CC
 from sage.rings.integer import Integer
 from sage.misc.decorators import options, rename_keyword
-from sage.symbolic.constants import pi, e
-from sage.functions.hyperbolic import arccosh
-from sage.functions.trig import sin, cos, cot
+from sage.misc.lazy_import import lazy_import
+lazy_import("sage.symbolic.constants", ["pi", "e"])
+lazy_import("sage.functions.hyperbolic", "arccosh")
+lazy_import("sage.functions.trig", ["sin", "cos", "cot"])
 from sage.misc.functional import is_odd
 from sage.matrix.constructor import matrix
 
```

Comment:
The same comment before: please separate out `lazy_import` statements from normal imports.

Reply:
Done.